lando_messaging/messaging.py

```diff
@@ -143,15 +143,16 @@ class StageJobPayload(object):
     """
     Payload that to be sent with JobCommands.STAGE_JOB to lando_worker.
     """
-    def __init__(self, credentials, job_id, input_files, vm_instance_name):
+    def __init__(self, credentials, job_details, input_files, vm_instance_name):
         """
-        :param credentials: jobapi.Credentials: keys used to download files
-        :param job_id: int: job id we want to have lando download files for
+        :param job_details: object: details about job(id, name, created date, workflow version)
```

Comment:
Looks like this `:param` got duplicated and you'd still want the one for credentials